src/components/profile/settings/user-card.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="profile-user-card">
+        <div class="d-flex mb-2">
+            <div class="profile-icon rounded-circle me-1">
+                <img src="/img/demo/my-profile.png" alt="Profile Image">
+            </div>
+            <h5 class="align-self-center text-light">{{ username }}</h5>
+            <div class="my-auto mx-0 align-self-center ms-1">
+                <PhCheck :size="32" color="#28bb94" weight="bold" v-if="isUserVerified" />
+                <PhWarning :size="32" color="#f31b65" weight="fill" v-else/>
```

Comment:
Make sure flyarrow `warning` color is added to `_fly_arrow_variables.scss`
```scss
$warning: #f31b65;
$warning-rgb: 243, 27, 101;
```
use `class="text-warning"` instead of `color="#f31b65"`